app/models/user.rb

```diff
@@ -16,4 +18,9 @@ def normalize_queued_videos_order
   def video_queued?(video)
     video ? (queued_videos & video.queued_videos).any? : false
   end
+
+  def has_follower?(other_user)
+    return false unless other_user
```

Comment:
Since the method requires the argument, this line should not be necessary. If there is no other user provided, you will get an ArgumentError before you hit this line.